server/src/main/java/keywhiz/service/resources/BatchSecretDeliveryResource.java

```diff
@@ -130,9 +131,11 @@ public BatchSecretDeliveryResource(@Readonly SecretController secretController,
     logger.info("Client {} granted access to {}.", client.getName(), clientAccessibleSecrets.stream().map(s -> s.name()).collect(toList()));
     try {
       // This is only possible if all secrets are both existing AND accessible to the client
-      return existingSecrets.stream()
+      List<SecretDeliveryResponse> secrets = existingSecrets.stream()
               .map(SecretDeliveryResponse::fromSecret)
               .collect(toList());
+      setTag("nSecrets", secrets.size());
```

Comment:
nit: should we say "numberOfSecrets" or "numSecrets" to make it clearer? I guess "nSecrets" isn't unclear, but why not spell it out in full?